agreement/msgp_gen.go

```diff
@@ -1211,9 +1211,6 @@ func (z period) MarshalMsg(b []byte) (o []byte, err error) {
 
 func (_ period) CanMarshalMsg(z interface{}) bool {
 	_, ok := (z).(period)
-	if !ok {
```

Comment:
Hm, my msgp isn't doing this -- can you please run `go get -u github.com/algorand/msgp` and then `make msgp` and see if it matches?

Reply:
you are right, seems like my msgp was outdated